docs/sources/basics/intro-histograms.md

```diff
@@ -8,8 +8,8 @@ weight = 700
 
 # Introduction to histograms and heatmaps
 
-A histogram is a graphical representation of the distribution of numerical data. It groups values into buckets
-(sometimes also called bins) and then counts how many values fall into each bucket.
+A histogram is a graphical representation of the distribution of numerical data. It groups values into _buckets_
+(sometimes also called _bins_) and then counts how many values fall into each bucket.
```

Comment:
bins and buckets do not need to be italicized.

Reply:
I wanted to emphasize it as it is a definition of a new term. But yes, I will revert that part